README.md

```diff
@@ -387,8 +396,10 @@ v3c.write(backend="kv", table="mytable", dfs=df, expression="city='NY'", conditi
 
 The following `write` parameters are specific to the `tsdb` backend; for more information about these parameters, see the [V3IO TSDB documentation](https://github.com/v3io/v3io-tsdb#v3io-tsdb):
 
-- <a id="method-write-param-labels"></a>**labels** (Optional) (default: `None`) &mdash; `dict` &mdash; A dictionary of `<label name>: <label value>` pairs that define metric labels to add to all written metric-sample table items.
-  Note that the values of the metric labels must be of type string.
+- <a id="method-write-tsdb-param-labels"></a>**labels** (Optional) (default: `None`) &mdash; `dict` &mdash;
+A dictionary of metric labels for the written item (row) of the format `{<label>: <value>[, <label>: <value>, ...]}`.
```

Comment:
Need to mention that this labels will apply for all rows in that DF so if you write `{"os": "linux"}` all rows will have the label os with the value linux, as apposed to specifying with index columns where you can specify different values for different rows, obviously.

Reply:
ACCEPT — I edited both the `dfs` and `labels` param descriptions for the `tsdb` backend, both in the README file and in the internal help text, together with the changes that I made for your [`dfs` review comment](https://github.com/v3io/frames/pull/306#discussion_r348461286). See commit 8f8fc8e.

Reply:
Approved by Tal